russian/zefir.md

```diff
@@ -119,3 +119,5 @@
 [Зайти на двач](2ch/2ch.md)
 
 [Стану вегетарианцем](vegetarian/vegetarian.md)
+
+[Открыть дверь шкафа в дивный мир](wardrobe-to-narnia/wardrobe-to-narnia.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.